### DISCUSSION TITLE
Schedule for the next release?

### DISCUSSION THREAD

**Discussion by nedsociety** [gitextensions/gitextensions#11573] [Q&A]
Is there any plan to release 4.2.2 or 4.3 in near future? I've encountered a problem that would be fixed by #11429 so it'd be great to try out the next RC 😄

**Comment by mstv** [comment]
My 2ct: I have added important bugfixes and a low-risk feature to [milestone 4.3](https://github.com/gitextensions/gitextensions/milestone/72). I would not merge much more than these into the main branch for a first beta with .NET 8 - "git grep" might become ready in time, too.
After branching off for v4.3, I want to merge the approved #11521 and to continue with the graph-rendering improvements.

**Reply by gerhardol** [reply]
master is currently in development phase, with many open PRs and additions. Nothing really major though. I would say that master is as stable as 4.2.1 (maybe even better, there are many corrections) but the plugin interface is being changed, there will need to be some time to get plugins ready. Maybe #11431 should be added too.
A roadblocker for any release is #11574 to fix the installer as it is blocked by AppVeyor.

**Comment by pmiossec** [comment]
>but the plugin interface is being changed,

I would like to see #11538 in next version because it introduced changes related to plugin interfaces and that could be good for few external plugins. So I prefer them to have to make both adaptations at the same time.

**Comment by totocaca123** [comment]
Is there any news about a potential release? Lots of changes occured since

**Reply by RussKie** [reply]
We need pull ourselves together but seems there's always something in the way.... :-\\
But I think we have less and less excuses :)

@gerhardol @mstv @pmiossec is there anything specific you want included in the next release?

**Reply by gerhardol** [reply]
Nothing major for me

**Reply by pmiossec** [reply]
Nothing important for me either.

My only concern is that I don't know if the plugin manager has been adapted to changes made to gitextensions Api. At a moment it was incompatible and I had to remove it and I still have not tested if it is OK now...

If someone has the answer, I'm interested: like that maybe I won't have to spend time on it...

**Reply by RussKie** [reply]
Looks like we're dependent on https://github.com/gitextensions/gitextensions.pluginmanager/pull/73.

**Reply by RussKie** [reply]
> Looks like we're dependent on [gitextensions/gitextensions.pluginmanager#73](https://github.com/gitextensions/gitextensions.pluginmanager/pull/73).

The PM has been updated (https://github.com/gitextensions/gitextensions.pluginmanager/releases/tag/v3.0.0-RC1). I've performed a test build, and the latest version of the PM is being bundled in: https://ci.appveyor.com/project/gitextensions/gitextensions/builds/50316520.
That build needs testing, as I haven't had time to perform any validation.

/cc: @maraf

**Reply by RussKie** [reply]
Here's a test build: https://ci.appveyor.com/project/gitextensions/gitextensions/builds/50328130/artifacts

**Reply by gerhardol** [reply]
Doc is updated with a 4.3 branch

**Reply by RussKie** [reply]
Here's the latest alpha build: https://ci.appveyor.com/api/buildjobs/6foesjdsu3dsks1g/artifacts/combined.5.0.0.17886-unsigned.zip

**Reply by RussKie** [reply]
@gitextensions/git-extensions-team can you please help with the summary for release note-worthy features you worked during the release --> https://github.com/gitextensions/gitextensions/releases

**Reply by mstv** [reply]
I have pre-filtered the list of revisions.

**Reply by RussKie** [reply]
Thank you, though I'm thinking more in terms of short summaries for the most note-worthy features. Users can browse the full list in their spare time.

**Reply by RussKie** [reply]
I like your breakdown into areas though 👍

**Reply by gerhardol** [reply]
Maybe https://github.com/gitextensions/gitextensions/pull/11484

**Comment by RussKie** [comment]
This may be a blocker for the release --> https://github.com/dotnet/winforms/issues/12003

**Reply by gerhardol** [reply]
Alpha for translators and plugin developers anyway?
(without freezing the interfaces yet)

**Reply by mstv** [reply]
> This may be a blocker for the release --> [dotnet/winforms#12003](https://github.com/dotnet/winforms/issues/12003)

Shouldn't be a blocker because we have silenced this exception in #11552.

> Alpha for translators and plugin developers anyway?

yes

> (without freezing the interfaces yet)

#11850 has been merged. Which other API changes are planned?

**Comment by RussKie** [comment]
v5.0 has been released.
